requirements/dev.in

```diff
@@ -1,3 +1,4 @@
 # Local development dependencies go here
 
 -r base.in                          # Core dependencies of edx-analytics-data-api
+mysqlclient
```

Comment:
Can you explain why this is needed now with a native docker file? Is it the case that Ansible was taking care of this dependency?

Reply:
Yeah, in ansible the production file was also being installed along with the dev requirements file. That's not what we want now as dev target should only install dev requirements file, but when I tried to do this the container threw errors while connecting to mysql because `mysqlclient` was missing. That's why I added it to `dev.in` instead of installing the `production.txt` in dev image.

Reply:
Thanks for that explanation. This make sense.